spec/graphql/cache/deconstructor_spec.rb

```diff
@@ -19,6 +19,18 @@ module Cache
           it 'should return raw.object' do
             expect(subject.perform).to eq raw.object
           end
+
+          context 'when obnject is a Graphql::Execution::Lazy whose value responsds to object' do
```

Comment:
Tiny typo here: `obnject` => `object`

Reply:
also `reponsds`. I swear I can type, lol. Thanks for the catch!